packages/mjml-spacer/src/index.js

```diff
@@ -40,7 +39,7 @@ export default class MjSpacer extends BodyComponent {
         <table role="presentation" border="0" cellpadding="0" cellspacing="0"><tr><td height="${parseInt(
           height,
           10,
-        )}" style="vertical-align:top;height:${height};">
+        )} " style="height:${height};">
```

Comment:
Can you remove the extra space in `)} "` -> `)}"`

Reply:
Done